netutils/interface.py

```diff
@@ -310,7 +324,7 @@ class CCSeparator(CharacterClass):
 
     weights: t.Dict[str, int] = {".": 10, "/": 20}
 
-    def __lt__(self, other) -> bool:  # noqa: D105
+    def __lt__(self, other) -> bool:  # type: ignore  # noqa: D105
```

Comment:
Can be fixed with `def __lt__(self, other: "CharacterClass") -> bool: # noqa: D105`